### ISSUE TITLE
[Bug] Do not check links that contain * signs

### ISSUE THREAD

**Issue by jiazengp** [lycheeverse/lychee-action#114]

**Comment by mre** [comment]
Hey @jiazengp, can you provide an example?

**Comment by mre** [comment]
As a workaround you could exclude these yourself for now by adding `--exclude '\*'`.

**Comment by jiazengp** [comment]
> 嘿，@jiazengp，你能举个例子吗？

For example

```html
<meta http-equiv="Content-Security-Policy" content="default-src 'none'; base-uri 'none'; connect-src https:; font-src 'self' data:  https://*.google.com;">
```

![image](https://user-images.githubusercontent.com/86162890/166112022-1a921fe1-47ea-4cde-9e43-a1f89c3713b4.png)

**Comment by mre** [comment]
Yeah that is indeed an issue. I think we should move the issue to the lychee repository, which is where we maintain the Rust code that lychee-action uses. In fact we should clarify with the team behind linkify, which is the plaintext link extractor we use. I'm not sure if this should count as a valid link (but it might very well be).
